datapipelines/src/test/java/io/customer/datapipelines/sdk/AndroidLifecyclePluginTests.kt

```diff
@@ -76,8 +76,9 @@ class AndroidLifecyclePluginTests : IntegrationTest() {
         val tracks = mutableListOf<TrackEvent>()
         verify { mockPlugin.track(capture(tracks)) }
         // 1. Application Installed
-        // 2. Application Opened
-        assertEquals(2, tracks.size)
+        // 2. Application Foregrounded
+        // 3. Application Opened
+        assertEquals(3, tracks.size)
```

Comment:
would we always get `opened` and `foregrounded`? because looking at the implementation they both are following the same strategy?